src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
```

Comment:
Don't use verbs for class/interface names. In this case, the class name 'IsoscelesTrapezoid' is appropriate as it's a noun, representing a geometric figure.